cluster/terraform_kubernetes/config/prometheus/datasources.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: 1
+datasources:
+- name: Thanos
+  type: prometheus
+  access: proxy
+  url: http://thanos-querier.monitoring.svc.cluster.local:9090
+  isDefault: true
+  jsonData:
+    timeInterval: "5s"
```

Comment:
default for httpMethod is POST, and this seems to be causing the HTTP 405's in the logs. Changing to GET stops these errors. So add,
httpMethod: GET in the jsonData: section (I think)